src/FixedDataTableNew.react.js

```diff
@@ -32,6 +32,7 @@ var invariant = require("invariant");
 var joinClasses = require("joinClasses");
 var shallowEqual = require("shallowEqual");
 var translateDOMPositionXY = require("translateDOMPositionXY");
+var throttle = require("lodash.throttle");
```

Comment:
@dennisberko Not that we're consistent in our app at all, but curious to hear your reasoning behind using "throttle" over "debounce". No preference on my side, just wanted to pick your brain. :)

Reply:
The key difference between debounce and throttle is that debounce waits until the events are done to fire where as throttle will fire periodically. When we want to trigger scroll handlers throttle is the appropriate function

Here is a neat demo that shows the difference between the two: http://demo.nimius.net/debounce_throttle/